src/platforms/px4_workqueue.h

```diff
@@ -46,6 +46,10 @@
 #include <stdint.h>
 #include <queue.h>
 
+#ifdef __PX4_QURT
```

Comment:
Can we move this to a platform types header file which resides in the platform directory and gets pulled in depending on which codebase we're building? The header could be empty for now for other platforms.